packages/odyssey-react-labs/package.json

```diff
@@ -12,6 +12,9 @@
     ".": {
       "default": "./dist/index.js",
       "types": "./dist/src/index.d.ts"
+    },
+    "./README.md": {
+      "default": "./README.md"
     }
```

Comment:
required to import from `packages/odyssey-storybook/src/installation/Odyssey-React-MUI.stories.mdx`